### PR TITLE
Proteore Venn diagram tool 

### PR DESCRIPTION
**Pull request by combesf**
Hello, 
Another ProteoRE tool for the review. 
Thanks !

### CODE REVIEW

**Comment by bgruening** [comment]
@combesf do you want to review these PRs gradually or when you have them all together?

You have a few things that apply to all tools.

Also do you know https://github.com/galaxyproject/galaxy-language-server ... this is very handy and helps you by formating the tools and catching some common errors.

**Comment by combesf** [comment]
@bgruening review the PR gradually is ok for us, if it is ok for you. 

Thanks for the tip, I'll take a look !

File: tools/proteore_venn_diagram/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+categories: [Proteomics]
+description: ProteoRE JVenn Diagram
+long_description: A tool for processing lists and produces Venn diagrams
+name: proteore_venn_diagram
+owner: proteore
```
Comment (bgruening) [inline tools/proteore_venn_diagram/.shed.yml:5]: galaxyp

File: tools/proteore_venn_diagram/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+include:
```
Comment (bgruening) [inline tools/proteore_venn_diagram/.shed.yml:6]: remove the include completely, or why do you use it?

File: tools/proteore_venn_diagram/.shed.yml (hunk 1 above)
Comment (yvandenb) [reply tools/proteore_venn_diagram/.shed.yml:5]: right: the ownership "proteore" is going to be replaced by "galaxyp" in each ".yml" tools asap...

**Review by bgruening — APPROVED** [review]

**Pull request merged**